lib/GaletteEvents/Controllers/CsvController.php

```diff
@@ -136,6 +137,7 @@ public function bookingsExport(Request $request, Response $response, array $args
         foreach ($bookings_list as $booking) {
             $member = $booking->getMember();
             $entry = [
+                $booking->getEvent()->getName(),
```

Comment:
It's technically OK, but this will just repeat the same (maybe long) name, and I'm not sure this is really accurate. I had in mind adding that as "a title" or something similar (I don't know if current exports permit something like that)

Reply:
> It's technically OK, but this will just repeat the same (maybe long) name, and I'm not sure this is really accurate.

Having at least the event id in a column would be very useful in case of a full export.

> I had in mind adding that as "a title" or something similar (I don't know if current exports permit something like that)

On a single line in the CSV file you mean ?
I'll have a look.

Reply:
I don't seen anything in `Galette\IO\CsvOut` that allows to add some kind of title in the CSV file :confused:

Reply:
Indeed, there is nothing existing yet; current PR is therefore OK as is; thanks :)